WooCommerce/Classes/POS/Presentation/POSReceiptEligibilityBanner.swift

```diff
@@ -29,7 +29,7 @@ struct POSReceiptEligibilityBanner: View {
 private extension POSReceiptEligibilityBanner {
     enum Constants {
         static let elementSpacing: CGFloat = 8
-        static let cornerRadius: CGFloat = 20
+        static let cornerRadius: CGFloat = POSCornerRadiusStyle.large.value
```

Comment:
20px radius is not in the design system, and I picked `large` for this case:

<img width="944" alt="Screenshot 2025-02-13 at 1 36 37 PM" src="https://github.com/user-attachments/assets/756665d1-002f-4726-a2b6-fb9d217f57bc" />